### PR TITLE
Add CDF TOC Contributors

### PR DESCRIPTION
**Pull request by oleg-nenashev**

### CODE REVIEW

**Comment by afrittoli** [comment]
Thanks @oleg-nenashev - it looks good to me. It's the idea then that each of us add our own details in a dedicated PR?

**Comment by oleg-nenashev** [comment]
@afrittoli The TOC members are listed here: https://github.com/cdfoundation/toc#members . This page is only for TOC Contributors (see the charter on https://github.com/cdfoundation/toc/blob/master/CONTRIBUTING.md#toc-contributors)

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -2,8 +2,10 @@
...
 
-__NOTE__: Add your name in the following format to the list below:
-
-```
-* First & Last name, Project/Company Affiliation (email)
-```
+Person | Organization | Other Affiliations | GitHub Username | Gitlab Username
+-- | -- | -- | -- | --
+Ankit Mohapatra | Dexai Robotics| Jenkins X maintainer | ankitm123 | 
+Kara de la Marck | CDF | Linux Foundation, CDF Interoperability SIG, Codebar | MarckK | |
```
Comment (ankitm123) [inline CONTRIBUTORS.md:8]: extra `|`.

**Comment by afrittoli** [comment]
> @afrittoli The TOC members are listed here: https://github.com/cdfoundation/toc#members . This page is only for TOC Contributors (see the charter on https://github.com/cdfoundation/toc/blob/master/CONTRIBUTING.md#toc-contributors)

:facepalm: Thanks @oleg-nenashev I should drink more coffee ☕ before commenting on GitHub issues 🙇

File: CONTRIBUTORS.md (hunk 1 above)
Comment (afrittoli) [reply CONTRIBUTORS.md:8]: I believe the extra column is for the GitLab username, it seems that GitHub md engine renders it ok whether it's specified empty or skipped

Comment (ankitm123) [reply CONTRIBUTORS.md:8]: We should add it to all rows I think or remove the extra `|` in the second row (I use markdownlint, not sure if that will catch this), but yes, it does render properly.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -2,8 +2,10 @@
...
 Find out more about the role of TOC contributors [here](CONTRIBUTING.md#toc-contributors)
 
-__NOTE__: Add your name in the following format to the list below:
-
-```
-* First & Last name, Project/Company Affiliation (email)
-```
+Person | Organization | Other Affiliations | GitHub Username | Gitlab Username
+-- | -- | -- | -- | --
+Ankit Mohapatra | Dexai Robotics| Jenkins X maintainer | ankitm123 |
```
Comment (ankitm123) [inline CONTRIBUTORS.md:7]: can we change this to `Ankit D Mohapatra`?

Comment (oleg-nenashev) [inline CONTRIBUTORS.md:7]:
```suggestion
Ankit D Mohapatra | Dexai Robotics| Jenkins X maintainer | ankitm123 | 
```

File: CONTRIBUTORS.md (hunk 1 above)
Comment (oleg-nenashev) [reply CONTRIBUTORS.md:8]: It was for Gitlab references, and in principle it should work. Will add to all entries

**Comment by ankitm123** [comment]
lgtm 💯

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -2,8 +2,10 @@
...
 
-__NOTE__: Add your name in the following format to the list below:
-
-```
-* First & Last name, Project/Company Affiliation (email)
-```
+Person | Organization | Other Affiliations | GitHub Username | Gitlab Username |
+-- | -- | -- | -- | -- |
+Ankit D Mohapatra | Dexai Robotics| Jenkins X maintainer | ankitm123 | |
+Kara de la Marck | CDF | Linux Foundation, CDF Interoperability SIG, Codebar | MarckK | |
```
[the remark this answers is not in the document]
Comment (oleg-nenashev) [reply CONTRIBUTORS.md:8]:
```suggestion
Kara de la Marck | CDF | Linux Foundation, CDF Interoperability SIG, codebar.io | MarckK | |
```

**Pull request merged**